Python/other/password_genretor.py

```diff
@@ -0,0 +1,29 @@
+import string
+import random
+if __name__ == '__main__':
+    # The lower case for password
+    accsi_1 = string.ascii_lowercase
+    # the upper case for password
+    accsi_2 = string.ascii_uppercase
+    # The numbers for password
+    digit_1 = string.digits
+    # The symbols for password
+    symbol1 = string.punctuation
+    password_len = int(input(("Entre lenght of the Password That You Want:")))
```

Comment:
Can you please shift the logical part to a new function and use the main function just for input/output purpose?

Reply:
ohk I do it.